docs/querying_history.rst

```diff
@@ -175,30 +175,49 @@ model history.
     <Poll: Poll object as of 2010-10-25 18:04:13.814128>
 
 
-Save without a historical record
---------------------------------
+Save Without Creating Historical Records
```

Comment:
Most titles don't use title casing in the file. I think you should reconsider this choice.

Reply:
Ah yes, I agree 👌 I was planning on polishing the docs in a later PR, which includes title-casing all the headings - to match both the main headings in the docs sidebar and various subheadings on some of the pages.

Any opinions on whether we should make title casing or normal casing the norm for the subheadings, btw?

Reply:
I prefer the normal casing for sentence titles here, but it's not something I'd argue to change an existing set of documentation over to.